gulpfile.js/index.js

```diff
@@ -7,26 +9,11 @@ const concat = require('gulp-concat');
 const replace = require('gulp-replace');
 const pump = require('pump');
 const fs = require('fs');
-const simpleGit = require('simple-git');
-const shelljs = require('shelljs');
 
-const paths = {
-	componentsFile: 'components.json',
-	componentsFileJS: 'components.js',
-	components: ['components/**/*.js', '!components/index.js', '!components/**/*.min.js'],
-	main: [
-		'components/prism-core.js',
-		'components/prism-markup.js',
-		'components/prism-css.js',
-		'components/prism-clike.js',
-		'components/prism-javascript.js',
-		'plugins/file-highlight/prism-file-highlight.js'
-	],
-	plugins: ['plugins/**/*.js', '!plugins/**/*.min.js'],
-	showLanguagePlugin: 'plugins/show-language/prism-show-language.js',
-	autoloaderPlugin: 'plugins/autoloader/prism-autoloader.js',
-	changelog: 'CHANGELOG.md'
-};
+const paths = require('./paths');
+const { premerge } = require('./premerge');
+const { changes, linkify } = require('./changelog');
```

Comment:
Maybe we should use `Object.assign(export, require('./premerge'), require('./changelog'));` here?

Reply:
If anything, it should be:
```suggestion
Object.assign(exports, require('./premerge'), require('./changelog'));
```

`export` is a keyword, `exports` is the CommonJS variable.

Reply:
Ahhh right. Small typo.

Reply:
Eh, I do like having all the commands listed at the bottom of the file together.

Reply:
I also like that all commands are visible at the bottom. This is basically a tradeoff between reducing redundancies and readability.

With the current style, you can see all tasks but have to write the name of each task 3 times (declaration at the source file + import here + export).
With the new style, you have to look at other source files to see all exported tasks but you only have to write the name of each task once.

So, what's better?
I actually don't mind either, so please decide.

Reply:
All at the bottom imo

Reply:
Alright!